optimize/backend/src/main/java/io/camunda/optimize/service/UIConfigurationService.java

```diff
@@ -102,6 +105,14 @@ public UIConfigurationResponseDto getUIConfiguration() {
     return uiConfigurationDto;
   }
 
+  private boolean isCamundaLicenseValid() {
+    return camundaLicenseService.isCamundaLicenseValid();
+  }
+
+  private String getCamundaEnvironment() {
```

Comment:
Why is this called environment? It is not really returning an environment is it?

Also, do we really need it here for Optimize? We already return the Optimize profile on line 66, why not reuse that?

Reply:
`environment` came from the API design discussion here: https://github.com/camunda/camunda/issues/20540. Right now it's either be `saas`,`self-managed`, and `unknown`, but a possible future state could include `hybrid`.

I'd prefer to keep it, as these two fields are an exact mirror of what the Unified REST API `/license` endpoint is. The `/license` endpoint will be used in the future once Optimize is part of the single-application. I'd want to keep the transition from these temporary Optimize-specifc fields to the `/license` endpoint as painless as possible